launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -73,6 +74,14 @@
   /** Logger name to use when launching a child process. */
   public static final String CHILD_PROCESS_LOGGER_NAME = "spark.launcher.childProcLoggerName";
 
+  private static final String LAUNCHER_INTERNAL_PORT = "spark.launcher.internal.port";
+
+  private static final String CHILD_PROCESS_LAUNCHER_INTERNAL_SECRET = "spark.launcher.internal.secret";
+
+  private static final String CHILD_PROCESS_LAUNCHER_STOP_FLAG = "spark.launcher.internal.stop.flag";
```

Comment:
Same thing here. You should use a better name, like the one you use below (`stopIfInterrupted`, which is also not really accurate).